lib/web/desktop.go

```diff
@@ -370,3 +379,103 @@ func proxyWebsocketConn(ws *websocket.Conn, wds net.Conn) error {
 	}
 	return trace.NewAggregate(retErrs...)
 }
+
+// createCertificateBlob creates Certificate BLOB
+// It has following structure:
+// 	CertificateBlob {
+// 		PropertyID: u32, little endian,
+// 		Reserved: u32, little endian, must be set to 0x01 0x00 0x00 0x00
+// 		Length: u32, little endian
+// 		Value: certificate data
+// 	}
+//
+func createCertificateBlob(certData []byte) []byte {
+	buf := new(bytes.Buffer)
+	buf.Grow(len(certData) + 12)
+	// PropertyID for certificate is 32
+	binary.Write(buf, binary.LittleEndian, int32(32))
+	binary.Write(buf, binary.LittleEndian, int32(1))
+	binary.Write(buf, binary.LittleEndian, int32(len(certData)))
+	buf.Write(certData)
+
+	return buf.Bytes()
+}
+
+func (h *Handler) desktopAccessScriptConfigureHandle(w http.ResponseWriter, r *http.Request, p httprouter.Params) (interface{}, error) {
+	tokenStr := p.ByName("token")
```

Comment:
Should we check that this is not empty and return an error early?

Reply:
That's a good point. I'll add the check for the empty string here.